towerlib/towerlib.py

```diff
@@ -1847,14 +1848,14 @@ def create_job_template(self,  # pylint: disable=too-many-arguments, too-many-lo
             raise InvalidProject(project)
         if playbook not in project_.playbooks:
             raise InvalidPlaybook(playbook)
+        if not all([credential, credential_type]):
+            self._logger.error('Both credential and credential type should be provided.')
+            raise InvalidCredential(credential)
         if all([credential, credential_type]):
```

Comment:
Since the above is a not all this could be an else clause.

Reply:
if one of the 2 parameters have values, the not all statement will catch that.
but if neither of the 2 parameters has values, then the code below shouldn't be executed, correct?

Reply:
If neither has values, it will still be caught by the above. The above reads, if not both values are set, blow up.

Reply:
Should I leave out the else entirely and de-indent?